src/components/ShortcutDrawer/Icons.android.tsx

```diff
@@ -3,6 +3,7 @@ import { StyleSheet } from 'react-native'
 import Icon from 'react-native-vector-icons/MaterialCommunityIcons'
 
 import { IconProps, IconsType } from './Icons.types'
+import COLORS from '../../themes/colors'
```

Comment:
The closest import should be at the bottom. Add this ESLint package, perhaps?

https://github.com/lydell/eslint-plugin-simple-import-sort/

I personally prefer newlines between each level of grouping, including level of import:

```
import Colors from '../../themes/colors

import etc from './Icon.types'
```

so if you can configure this rule like that, even better.

Bunching relative imports together can become overwhelming imo

Reply:
https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/order.md

A different plugin, but I think the other might be better